src/zkevm_specs/evm/execution/call.py

```diff
@@ -151,7 +151,7 @@ def call(instruction: Instruction):
             (CallContextFieldTag.StackPointer, instruction.curr.stack_pointer + 6),
             (CallContextFieldTag.GasLeft, instruction.curr.gas_left - gas_cost - callee_gas_left),
             (CallContextFieldTag.MemorySize, next_memory_size),
-            (CallContextFieldTag.StateWriteCounter, instruction.curr.state_write_counter + 1),
+            (CallContextFieldTag.StateWriteCounter, instruction.curr.reversible_write_counter + 1),
```

Comment:
This should also be renamed.

```suggestion
            (CallContextFieldTag.ReversibleWriteCounter, instruction.curr.reversible_write_counter + 1),
```

Reply:
thanks, i've updated those.